RNTester/package.json

```diff
@@ -15,8 +16,7 @@
   },
   "devDependencies": {
     "babel-jest": "20.0.0",
-    "babel-preset-react-native": "1.9.1",
-    "fs-extra": "^3.0.1"
+    "babel-preset-react-native": "1.9.1"
```

Comment:
Why do we need babel transforms at all?

Reply:
we need this to build the iOS/Android app correctly :)